output_5.1.txt

```diff
@@ -0,0 +1,70 @@
+TASK: 4.2
```

Comment:
+ incorrect task name (this is not 4.2)
+ I do not understand what task number you are actually implementing here - the queries do not match any task